src/kata6.joinNames.js

```diff
@@ -1,3 +1,7 @@
-const joinNames = (namesObj) => {};
+const joinNames = (namesObj) => {
+  const namesArr = namesObj.map(({ name }) => name);
+  const lastName = namesArr.pop();
+  return `${namesArr.join(", ")} & ${lastName}`;
+};
```

Comment:
This function works for the test provided, but if you pass an array with only 1 name you get the wrong output, i.e. `joinNames([{ name: "Bart" }])` return `" & Bart"`